src/System.IO.FileSystem.Watcher/src/System/IO/FileSystemWatcher.Win32.cs

```diff
@@ -27,7 +27,7 @@ private void StartRaisingEvents()
 
             // Create handle to directory being monitored
             var defaultSecAttrs = default(Interop.Kernel32.SECURITY_ATTRIBUTES);
-            _directoryHandle = Interop.Kernel32.CreateFile(
+            _directoryHandle = Interop.Kernel32.UnsafeCreateFile(
```

Comment:
why this change?

Reply:
UnsafeCreateFile is the wrapper used to redirect between Win32 and Uap implementation. This is the approach used in System.IO.FileSystem

Reply:
oh, it's to abstract createfile/createfile2. I was confused because of the name of this file. the file should be FileSystemWatcher.Windows.cs, I think? Win32 means not UAP.

Reply:
Good point.